spec/requests/internal/users_spec.rb

```diff
@@ -29,7 +29,6 @@ def dependents_for_offending_user_article
     create(:reaction, reactable: comment2, reactable_type: "Comment", user: user2)
     # create user3 reaction to offending article
     create(:reaction, reactable: article, reactable_type: "Article", user: user3, category: "like")
-    Mention.create_all_without_delay(comment2)
```

Comment:
I suppose we still need mentions to create. You can use `Mention.create_all` with `perform_enqueued_jobs` or alternative methods to create them.